### PR TITLE
Force invariant culture for some ColorTranslator tests

### PR DESCRIPTION
**Pull request by mellinoe**
These tests fail on some cultures, including Polish. This change forces us to use InvariantCulture for these tests.

@hughbe @norek

### CODE REVIEW

File: src/System.Drawing.Common/tests/ColorTranslatorTests.cs [hunk 1]
```diff
@@ -188,7 +190,16 @@ public static IEnumerable<object[]> FromHtml_TestData()
         [MemberData(nameof(FromHtml_TestData))]
         public void FromHtml_String_ReturnsExpected(string htmlColor, Color expected)
         {
-            Assert.Equal(expected, ColorTranslator.FromHtml(htmlColor));
+            CultureInfo originalCulture = Thread.CurrentThread.CurrentCulture;
```
Comment (danmosemsft) [inline src/System.Drawing.Common/tests/ColorTranslatorTests.cs]: Would it be overkill to add some helper function centrally that takes a callback and a culture it should be run with?

Comment (mellinoe) [inline src/System.Drawing.Common/tests/ColorTranslatorTests.cs]: I've done this a couple times before, but it's not incredibly common. So I'd say no, but it wouldn't really hurt anything either way.

Comment (hughbe) [inline src/System.Drawing.Common/tests/ColorTranslatorTests.cs]: We have this test class: https://github.com/dotnet/corefx/blob/master/src/Common/tests/System/ThreadCultureChange.cs

Might be worth adding a ctor for the culture to it

Comment (mellinoe) [inline src/System.Drawing.Common/tests/ColorTranslatorTests.cs]: I added some helpers to `ThreadCultureChange` and am using it now.

File: src/Common/tests/System/ThreadCultureChange.cs [hunk 2]
```diff
@@ -17,13 +17,29 @@ public ThreadCultureChange()
...
+        {
+            ChangeCultureInfo(newCulture);
+        }
+
         public void ChangeCultureInfo(string culture)
         {
             var newCulture = new CultureInfo(culture);
             CultureInfo.CurrentCulture = newCulture;
             CultureInfo.CurrentUICulture = newCulture;
         }
```
Comment (stephentoub) [inline src/Common/tests/System/ThreadCultureChange.cs]: This method could now just be:
```C#
public void ChangeCultureInfo(string culture) => ChangeCultureInfo(new CultureInfo(culture));
```

File: src/Common/tests/System/ThreadCultureChange.cs [hunk 3]
```diff
@@ -17,13 +17,29 @@ public ThreadCultureChange()
             _originalUICultureInfo = CultureInfo.CurrentUICulture;
         }
 
+        public ThreadCultureChange(string culture) : this()
+        {
+            ChangeCultureInfo(culture);
+        }
+
+        public ThreadCultureChange(CultureInfo newCulture) : this()
```
Comment (stephentoub) [inline src/Common/tests/System/ThreadCultureChange.cs]: You could eliminate both the need for these `: this()` by changing the above from:
```C#
private readonly CultureInfo _originalCultureInfo;
private readonly CultureInfo _originalUICultureInfo;

public ThreadCultureChange()
{
    _originalCultureInfo = CultureInfo.CurrentCulture;
    _originalUICultureInfo = CultureInfo.CurrentUICulture;
}
```
to instead be:
```C#
private readonly CultureInfo _originalCultureInfo = CultureInfo.CurrentCulture;
private readonly CultureInfo _originalUICultureInfo = CultureInfo.CurrentUICulture;

public ThreadCultureChange() { }
```

**Pull request merged**